test/distributed/test_inductor_collectives.py

```diff
@@ -406,6 +406,44 @@ def test_meta(self):
         out = torch.ops.c10d_functional.all_reduce(x, "sum", **self.get_world_trs())
         assert x.size() == out.size()
 
+    @unittest.skipIf(not has_triton(), "Inductor+gpu needs triton and recent GPU arch")
+    @patch.object(torch._inductor.config.triton, "descriptive_names", False)
+    def test_inductor_all_gather_coalesced(self):
```

Comment:
was this test case meant to be on the next PR? or i'm confused

Reply:
yeah. fixing it.